packages/oui-collapsible/src/collapsible.html

```diff
@@ -1,11 +1,12 @@
 <button class="oui-button oui-button_full-width oui-collapsible__header"
     type="button"
-    ng-click="$ctrl.toggle()"
+    ng-click="!$ctrl.loading && $ctrl.toggle()"
```

Comment:
The button is not disabled ?

Reply:
No, we just disable the click during a loading. IMO, the disabled state should be something different. But both has the same effect (unclickable button) :)